substra/sdk/backends/local/backend.py

```diff
@@ -254,7 +254,7 @@ def __format_for_leaderboard(self, testtuple):
         algo = self._db.get(traintuple.algo_type, traintuple.algo.key)
         return {
             'algo': {
-                'hash': algo.key,
+                'key': algo.key,
```

Comment:
shouldn't there be a checksu in this object since there also is a storage_address ?